package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mattermost-mobile",
-  "version": "1.31.0",
+  "version": "1.30.0",
```

Comment:
I think you dont want to include this change

Reply:
Good point! Will fix 👍

Reply:
Done!